ios/StatusIm.xcodeproj/project.pbxproj

```diff
@@ -187,7 +186,6 @@
 			isa = PBXGroup;
 			children = (
 				2028E0111D4275BD00227DCD /* SF */,
-				B23B48FE1E76917B006D4535 /* RobotoMono-Medium.ttf */,
```

Comment:
@Ferossgp are you sure we don't use this anymore? we use monospaced fonts in places like account addresses, I guess all of them use the default 'monospace' declaration which defaults to 'Menlo' iOS. Not ideal, Roboto Mono would be better since it looks more similar to Inter. Even better would be SF Mono, the font Apple uses on iOS but afaik it's private and unavailable?

Reply:
good point

Reply:
Actually, the font does not exist, it was removed before that PR, I've removed the link to the inexistent font. I also found now that there is also a link to an inexisting SF font. We can replace the font to the one you prefer in a separate PR.

Reply:
Ok. I'll follow up on the monospace font declaration then in a separate issue. As for SF, thanks! For a minute there I was worried we still have it. It's licensed by Apple so we actually can't embed it our app assets, a day before the App Store launch we could get into some shit by doing that :)